decisions/fill-rule-in-svgs.md

```diff
@@ -0,0 +1,11 @@
+# We can't use SVGs that include `fill-rule`
+
+Whenever our icons contain the `fill-rule` property, they don't render correctly on iOS. This is because the process we use to convert SVGs into an icon font seems to strip out `fill-rule`.
```

Comment:
This is _**technically**_ wrong as they could use `fill-rule` unnecessarily. This might be more technically correct
```suggestion
Whenever our icons rely on `fill-rule="evenodd"`, they don't render correctly on iOS. This is because the process we use to convert SVGs into an icon the `fill-rule` is not applied.
```